### DISCUSSION TITLE
Probleme d'affichage en mode Edition de parcours

### DISCUSSION THREAD

**Discussion by Superbatterie** [chamilo/chamilo-lms#5734] [General]
Bonjour à tt le monde,

Je suis débutant en Chamilo
J'ai installé Chamilo via les app d'Infomaniak sur mon serveur dans un rep /ecole

En mode édition du parcours la fenetre s'affiche comme si le frame faisait 7000 px de large .. (voir screen shots)
<img width="1525" alt="Capture d’écran 2024-08-17 à 20 41 39" src="https://github.com/user-attachments/assets/945f6f8b-9cd3-4ece-8075-1722a4b283f6">
<img width="1230" alt="Capture d’écran 2024-08-17 à 20 43 44" src="https://github.com/user-attachments/assets/6c3b09c2-7391-46d4-9957-7dd894df7bae">
ça fait beaucoup ^^

Quelqu'un sait-il comment remet-on ça en face ? (et si elle/il peut m'aiguiller pour le faire avant la rentrée c'est génial)

Merci !!

**Comment by NicoDucou** [comment]
Bonjour
Nous avons remarqué ce problème avec un Chamilo installé sur un serveur avec PHP8.3
Il semblerait que cela correspond au problème qui a été solutionné par les commits suivants : 
- 377b664622ed6bae06eb3746b1a72b0e28e6a756
- 29012184c5c6464b39e309ed4ae4bd6518ef6da5
- 18c9bed0d77807f2c1453da0bc4a83739144ea3b
- df773154da894f868d3f33ad1bf85fa0fc0ce1a5
- c31e220b41c6d6d8969cb909b9efada89fb84df0

Ces corrections sont incluses dans la prochaine version de Chamilo 1.11.28 qui sortira plus tard quand elle sera prête.
Donc si vous avez les compétences vous pouvez appliqué manuellement les changements correspondant à ces commits.

Sinon vous pouvez essayer de voir pour que votre hébergement soit avec php 7.4 version de PHP qui est bien supporté par la version de Chamilo que vous utilisez.

Sinon attendre que la version 1.11.28 de Chamilo sorte mais il n'y a pas encore de date exacte pour sa sortie.

Cordialement

**Reply by Superbatterie** [reply]
Bonjour NicoDoou

Merci pour cette réponse très complète, j'ai beaucoup de matière à potasser c'est dense, j'avoues que je n'avais pas regarder les sujets en anglais, je n'ai pas le vocabulaire.
Ma version php proposée est PHP8.2 si cela peut aider.
Je vais donc lire tout ça et voir si je suis compètent.

Bien à vous
